app/src/main/res/values/strings.xml

```diff
@@ -19,11 +18,11 @@
     <string name="account">Account</string>
     <string name="quick_picks">Quick picks</string>
     <string name="quick_picks_empty">Listen to songs to generate your quick picks</string>
-    <string name="new_release_albums">New releases</string>
     <string name="forgotten_favorites">Forgotten favorites</string>
-    <string name="similar_to">Similar to</string>
     <string name="keep_listening">Keep listening</string>
-    <string name="your_ytb_playlists">Your YouTube playlists</string>
+    <string name="your_youtube_playlists">Your YouTube playlists</string>
```

Comment:
your_ytb_playlists was replaced with your_youtube_playlists which is the same thing but available upstream